frontend/src/v5/ui/routes/dashboard/projects/tickets/ticketsTable/ticketsTable.component.tsx

```diff
@@ -143,9 +133,10 @@ export const TicketsTable = () => {
 		return pathname + (ticketId ? `?ticketId=${ticketId}` : '');
 	};
 
+	useSetDefaultTicketFilters([selectedTemplate]);
```

Comment:
Here I think it should be `getNonCompletedTicketFilters`   but as a hook.
`useSetDefaultTicketFilters` currently uses the routing for the viewer.

So rename `getNonCompletedTicketFilters` to `useSetDefaultTicketFilters`

Rename `useViewerTicketFilterParams` to `useBindTicketFiltersData` make it receive any param like the one from needed in the tickets table to reuse that in both places.  Add any necessary logic  (like the filtersInitialised)  the idea is to have a hook that orchestrates how tickets filters data is fetched.
 

and rename 
`useSetDefaultTicketFilters` to `useViewerTicketFilterParams`